plugin/core/protocol.py

```diff
@@ -327,7 +327,7 @@ class DocumentHighlightKind(IntEnum):
     """ Write-access of a symbol, like writing to a variable. """
 
 
-class CodeActionKind(Enum):
+class CodeActionKind(Enum, str):
```

Comment:
In Typescript terms, the spec seems to be merging declaration of `type` and `namespace`. I wasn't sure if there is an equivalent of that in Python types. If `class CodeActionKind(str, Enum)` doesn't have any drawbacks then I guess I would be fine with that.

I can't tell from the top of my head how to fix that in lsp-python-types but I guess it would have to be a special case like https://github.com/sublimelsp/lsp-python-types/blob/c27bb7263737a0d2b1d6840dbef3b92db7bb1252/generate.py#L11-L11 which would affects relevant code.

Reply:
Another alternative:
```python
from enum import StrEnum
```
from Python 3.11.

Are we able to use this?

Reply:
Yes, we are. We are depending on 3.11 version already for NonRequired and stuff.

Reply:
Created https://github.com/sublimelsp/lsp-python-types/pull/10